src/components/body/body-cell.component.ts

```diff
@@ -225,13 +226,21 @@ export class DataTableBodyCellComponent implements DoCheck, OnDestroy {
       const userPipe: PipeTransform = this.column.pipe;
 
       if (userPipe) {
-        value = userPipe.transform(val);
+        let transformResult = userPipe.transform(val);
+        if (transformResult instanceof Observable) {
+          let observable: Observable<any> = transformResult;
+          observable.subscribe((transformedValue) => {
```

Comment:
You never unsubscribe from this, I think this would create a memory leak.

Reply:
What do you think about something like this:

let pipeSubscription = observable.subscribe((transformedValue) => {
      value = transformedValue;
      pipeSubscription.unsubscribe();
 });